PocketKit/Sources/PocketKit/Premium/PremiumUpgradeView.swift

```diff
@@ -175,12 +176,51 @@ Refunds are not available for unused portions of a subscription.
 }
 
 private struct PremiumTermsView: View {
+    @State var showPrivacyPolicy = false
+    @State var showTermsOfService = false
+
     var body: some View {
         HStack(spacing: 16) {
-            Button { } label: { Text("Privacy Policy").style(.terms) }
-            Button { } label: { Text("Terms of Service").style(.terms) }
+            Button(action: {
+                self.showPrivacyPolicy = true
+            }, label: { Text("Privacy Policy").style(.terms) })
+            .sheet(isPresented: $showPrivacyPolicy) {
+                if let privacyUrl = getUrlFor(typeOf: .PrivacyPolicy) {
+                    SFSafariView(url: privacyUrl)
+                }
+            }
+            .accessibilityIdentifier("privacy-policy")
+            Button(action: {
+                self.showTermsOfService = true
+            }, label: { Text("Terms of Service").style(.terms) })
+            .sheet(isPresented: $showTermsOfService) {
+                if let toSUrl = getUrlFor(typeOf: .TermsOfService) {
+                    SFSafariView(url: toSUrl)
+                }
+            }
+            .accessibilityIdentifier("terms-of-service")
         }.padding(.bottom)
     }
+
+    enum Link {
+        case PrivacyPolicy
+        case TermsOfService
```

Comment:
nit: enum cases usually lower camel case
https://github.com/kodecocodes/swift-style-guide#naming

Reply:
@cyndichin can we enforce this with Swiftlint in another PR? Seems like we should.

Reply:
https://getpocket.atlassian.net/jira/software/projects/IN/boards/80/backlog

Reply:
hmmm I don't see a SwiftLint rule for this, but we're planning on having a discussion relating to this. 

https://pocket.slack.com/archives/C01UH57EJKD/p1676391464059509

This is what I'm used to in Swift convention, but happy to revisit as a team.

```
## Naming

### Enum
When it comes to enum types, we like to use `UpperCamelCase`.
When it comes to enum cases, we like to use `lowerCamelCase`.

```
enum ReadableEvent {
    case archive
    case delete
    case contentUpdated
}
```